content/en/core/releases/4.16.0.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "4.16.0 release notes"
+linkTitle: "4.16.0"
+weight:
+description: >
+relevantLinks: >
+toc_hide: true
+---
+
+## Known issues
+
+Check the repository for the [latest known issues](https://github.com/medic/cht-core/issues?q=is%3Aissue+label%3A%22Affects%3A+4.16.0%22).
+
+## Upgrade notes
+
+### Breaking changes
+
+None.
+
+### UI/UX changes
+
+None.
+
+## Highlights
+
+### Upgrading to CouchDb 3.4.2
+
+CouchDb 3.4.2 comes bundled by default with CHT-Core 4.16.0 and includes a multitude of improvements, including:
```

Comment:
```suggestion
CouchDb 3.4.2 comes bundled by default with CHT Core 4.16.0 and includes a multitude of improvements, such as:
```

_nitpick_: the `include` word is used twice in the sentence; replacing it with something else.